pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -14,9 +14,9 @@ stdenv.mkDerivation rec{
 
   src = fetchgit {
     url = "https://github.com/hrydgard/ppsspp.git";
-    rev = "8c8e5de89d52b8bcb968227d96cbf049d04d1241";
+    rev = "6d0d36bf914a3f5373627a362d65facdcfbbfe5f";
```

Comment:
Looks like I have to explicitly put "refs/tags/v$version". Is that expected? Has that changed recently?